src/graphql/dataLoaders/index.ts

```diff
@@ -24,6 +25,7 @@ const LOADER_FACTORY_MAP = {
   repliedArticleCountLoader: repliedArticleCountLoaderFactory,
   votedArticleReplyCountLoader: votedArticleReplyCountLoaderFactory,
   userLoader: userLoaderFactory,
+  badgeLoader: badgeLoaderFactory,
```

Comment:
Seems that `badgeLoader` is not used at all; generic `docLoader` is sufficient for loading badges in user field resolvers.

In this case we can remove `badgeLoader` and its implementations.